app/assets/stylesheets/all/datatables-bootstrap.scss

```diff
@@ -11,6 +11,7 @@
     // similar effect, but also adds unnecessary bottom padding which looks a
     // little clunky
     @extend .mr-3;
+    @extend .mt-3;
```

Comment:
The search box was butting uncomfortably against the top border, so just added a little more margin.